sites/public/styles/overrides.scss

```diff
@@ -232,3 +232,14 @@ a.button.is-primary:hover {
     border-left-width: 0;
   }
 }
+
+.eligibility-disclaimer-header {
+  padding: 20px 4rem;
+}
+
+.eligibility-disclaimer-submit-container {
+  text-align: center;
+  @screen sm {
+    padding-bottom: 40px;
```

Comment:
I think you might want this to be in the default css and then have padding-bottom: 0 for @screen sm? Basically on mobile I'm seeing that it needs padding but it doesn't at larger sizes

Reply:
Ya, not sure what I was thinking here.